testing/spectest/shared/common/forkchoice/builder.go

```diff
@@ -101,15 +101,15 @@ func (bb *Builder) block(t testing.TB, b interfaces.ReadOnlySignedBeaconBlock) [
 // InvalidBlock receives the invalid block and notifies forkchoice.
 func (bb *Builder) InvalidBlock(t testing.TB, b interfaces.ReadOnlySignedBeaconBlock) {
 	r := bb.block(t, b)
-	ctx, cancel := context.WithTimeout(context.TODO(), time.Second)
+	ctx, cancel := context.WithTimeout(context.TODO(), 3*time.Second)
```

Comment:
```suggestion
	ctx, cancel := context.WithTimeout(t.Context(), 3*time.Second)
```

Reply:
Why switch from 1 second to 3 seconds?
What about the Preston's suggestion?